src/Umbraco.Web/Mvc/RenderRouteHandler.cs

```diff
@@ -279,10 +280,40 @@ internal virtual RouteDefinition GetUmbracoRouteDefinition(RequestContext reques
                     //set the controller and name to the custom one
                     def.ControllerType = controllerType;
                     def.ControllerName = ControllerExtensions.GetControllerName(controllerType);
+
                     if (def.ControllerName != defaultControllerName)
                     {
                         def.HasHijackedRoute = true;
                     }
+
+                    // Look for hijacked action
+                    var hijackActions = controllerType
+                        .GetMethods(BindingFlags.Public | BindingFlags.Instance)
+                        .Select(x => Tuple.Create(x, x.GetCustomAttribute<HijackRouteAttribute>(true)))
```

Comment:
The `true` parameter here will mean that it will look into base classes/inherited structures which i really don't think you would want in this case? This will also add performance overhead.

Reply:
Yea, I think we can remove it and if someone wants to inherit a base controller they will have to override the child action and add the attribute on the override 👍